src/Simulation/NativeSparseSimulator/CMakeLists.txt

```diff
@@ -1,13 +1,24 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
+
+# Load common utils and configure cmake policies
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/../../Common/cmake")
+include(secure_dependencies)
+set_msvc_static_runtime_policy()
+
 project(Microsoft.Quantum.SparseSimulator.Runtime)
 
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
 set(CMAKE_MACOSX_RPATH 1)
+
+locate_win32_spectre_static_runtime()
+configure_security_flags()
+
 # Main build files
 add_library(Microsoft.Quantum.SparseSimulator.Runtime SHARED factory.cpp capi.cpp)
+target_link_libraries(Microsoft.Quantum.SparseSimulator.Runtime ${SPECTRE_LIBS})
```

Comment:
I assume that 
`${SPECTRE_LIBS}` expands to _nothing_ on non-Win platforms (because `SPECTRE_LIBS` is defined inside of `if (WIN32)` and on other platforms it stays undefined).

Reply:
That is correct, making the line a no-op. This matches the behavior we use for the other native binaries.